radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/RadixHash.java

```diff
@@ -4,6 +4,7 @@
 import java.nio.ByteBuffer;
 import java.util.Arrays;
 
+import okio.ByteString;
```

Comment:
Not really happy with this dependency being here.  Can we use `Bytes.toHexString(...)`?